pkg/ccl/backupccl/backupinfo/manifest_handling.go

```diff
@@ -97,6 +97,16 @@ var WriteMetadataSST = settings.RegisterBoolSetting(
 	util.ConstantWithMetamorphicTestBool("write-metadata-sst", false),
 )
 
+// WriteMetadataWithFilesSST controls if we write a `BACKUP_METADATA` file
+// along with an SST containing the list of `BackupManifest_Files`. This new format
+// of metadata is
+var WriteMetadataWithFilesSST = settings.RegisterBoolSetting(
+	settings.TenantWritable,
+	"backup.write_metadata_with_files_sst.enabled",
```

Comment:
should we change this to a more generic `backup.write_metadata_with_external_ssts.enabled`? Since we have more than just Files as SSTs now.

Reply:
updated the setting name and the doc comment